spec/models/post_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let (:post) { build(:post) }
+  let (:user) { build(:user) }
+
+  describe '#attributes' do
```

Comment:
Why did we have to write specs for `attributes` which is a rails built in method?